CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
  - add frontend test for draft-js editor
  - add a ribbon icon when an instructor is the writer of a post or a topic
+ - add feature in draftjs editor to mention any active user in the current topic
```

Comment:
```suggestion
 - add feature in AshleyEditor to mention any active user in the current topic
```
I suggest to use the name of our component instead of the technology it's based on.

And BTW, I realize that we should standardize the way we spell `Draft.js` in the Changelog (there is `Draft.js`, `draftjs`, `draft-js`). The official name of the project is `Draft.js`.